### PR TITLE
SAK-40185 Add details to closed view of rubric

### PR DESCRIPTION
**Pull request by adrianfish**

### CODE REVIEW

File: rubrics/tool/src/main/frontend/sass/sakai-rubrics.scss [hunk 1]
```diff
@@ -40,7 +45,25 @@
...
+#site-rubrics-title {
+  display: flex;
+  justify-content: space-between;
+  div {
+    font-weight: bold;
+    flex: 1;
+  }
+}
+
+#shared-rubrics-title {
```
Comment (ottenhoff) [inline rubrics/tool/src/main/frontend/sass/sakai-rubrics.scss]: if these two sections are identical, just consolidate to one rule?

File: rubrics/tool/src/main/frontend/imports/sakai-rubrics-styles.html [hunk 2]
```diff
@@ -33,7 +37,21 @@
...
+    width: 100px; }
+
+#site-rubrics-title {
+  display: flex;
+  justify-content: space-between; }
+  #site-rubrics-title div {
+    font-weight: bold;
+    flex: 1; }
+
+#shared-rubrics-title {
```
Comment (ottenhoff) [inline rubrics/tool/src/main/frontend/imports/sakai-rubrics-styles.html]: these two sections look identical

File: rubrics/api/src/main/java/org/sakaiproject/rubrics/logic/model/Metadata.java [hunk 3]
```diff
@@ -27,6 +27,8 @@
 import javax.persistence.Embeddable;
 import javax.persistence.Transient;
 
+import org.sakaiproject.rubrics.logic.RubricsLocalDateTimeSerializer;
```
Comment (ern) [inline rubrics/api/src/main/java/org/sakaiproject/rubrics/logic/model/Metadata.java]: Curious why dates are not sent using the ISO format and then converted to whatever is needed on the client.

Comment (adrianfish) [reply rubrics/api/src/main/java/org/sakaiproject/rubrics/logic/model/Metadata.java]: I don't know why, either. All I know is that they were being serialized in standard ISO format and that's how the ui code gets them. Alternatively, we could send them as epoch dates and let client side JS deal with it.

**Comment by ern** [comment]
While there are some questions they're mostly minor so going to merge this before the branch.

**Pull request merged**